components/webhook/Form.tsx

```diff
@@ -86,6 +86,7 @@ const Form = ({
                   <EventTypes
                     onChange={formik.handleChange}
                     values={initialValues['eventTypes']}
+                    error={formik.errors.eventTypes}
```

Comment:
@jatingodnani Can we keep the error message on single line?

Reply:
It is single line you can check

Reply:
![BoxyHQ (2)](https://github.com/boxyhq/saas-starter-kit/assets/4593041/9bebe163-09cb-49b3-82ae-e72204801122) See this screenshot

Reply:
Ok